testing/clazy/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+pkgname=clazy
+pkgver=1.5
+pkgrel=0
+pkgdesc="Qt oriented code checker based on clang framework"
+url="https://github.com/KDE/clazy"
+arch="all"
+license="GPLv2"
```

Comment:
https://spdx.org/licenses/

Please choose the correct identifier from above list.